src/views/Hearing/index.js

```diff
@@ -128,12 +145,12 @@ class Hearing extends React.Component {
     const hearingAllowsComments = !hearing.closed && (new Date() < new Date(hearing.close_at));
     const onPostVote = this.onVoteComment.bind(this);
     const introSection = find(hearing.sections, (section) => section.type === "introduction");
-    const closureInfoSection = find(hearing.sections, (section) => section.type === "closure-info");
+    const closureInfoSection = this.getClosureInfo();
```

Comment:
Maybe just pass `hearing` in as a param instead of re-deriving it in the helper? :)